_component_design/popovers.md

```diff
@@ -5,90 +5,180 @@ secondary: Editing
 layout: component
 category: Components
 usage: >
-  A popover is a small group of information or inputs that is presented in a panel that is overlayed on top of the original page, preserving the user’s context.
+  To preserve the user’s context, popovers present a small group of information or inputs that is presented in a panel that is overlayed on top of the original page, preserving the user’s context.
```

Comment:
Popovers present a small group of information or inputs that is presented in a panel. To preserve the user’s context, the panel is overlaid on top of the original page.

Reply:
Updated.